### PR TITLE
Changing Jupyter service to ClusterIP by default.

### PR DESCRIPTION
**Pull request by elsonrodriguez**

### CODE REVIEW

**Comment by jlewi** [comment]
LGTM but the code needs to be sync'd.

File: kubeflow/core/jupyterhub.libsonnet [hunk 1]
```diff
@@ -186,7 +186,7 @@ c.KubeSpawner.pvc_name_template = 'claim-{username}{servername}'
 	  }
    },
 
-   jupyterHubLoadBalancer: {
+   jupyterHubLoadBalancer(serviceType): {
```
[the remark this answers is not in the document]
Comment (elsonrodriguez) [reply kubeflow/core/jupyterhub.libsonnet]: That's a fair point. In the ClusterIP configuration `jupyterHubLoadBalancer` is almost a duplicate of the existing `jupyterHubService`. I don't know the reasoning behind having two services for Jupyter, but if someone could give me some pointers i could just put a flag into ksonnet whether or not to deploy `jupyterHubLoadBalancer` in the first place.

[the remark this answers is not in the document]
Comment (jlewi) [reply kubeflow/core/jupyterhub.libsonnet]: See https://github.com/google/kubeflow/issues/99 for why two services.

**Comment by jlewi** [comment]
I went ahead and merged this since the PR as is fixed the issue; feel free to send another PR if think changing the names is worth cleaning up.

**Pull request merged**